community_images/httpd/bitnami/module_gen.py

```diff
@@ -0,0 +1,32 @@
+import argparse
```

Comment:
where is this script getting used?

Reply:
module_gen.py was created to create a list of all the modules and add them to a file. Further, we could use the generated file by directly copying it's content tot he httpd.conf file. But, python didn't work after the stubbing was initiated. Therefore, I kept the script there for future use.

Reply:
Removed module_gen.py since it won't be used in this image. Keeping it safe for future use.